src/python/google_cloud_utils/storage.py

```diff
@@ -325,19 +337,23 @@ def _get_object_properties(self, remote_path):
     bucket, path = get_bucket_name_and_path(remote_path)
     fs_path = self.convert_path(remote_path)
 
-    return {
-        'bucket':
-            bucket,
-        'name':
-            path,
-        'updated':
-            datetime.datetime.utcfromtimestamp(os.stat(fs_path).st_mtime),
-        'size':
-            os.path.getsize(fs_path),
-        'metadata':
-            self._get_metadata(bucket, path),
+    data = {
+        'bucket': bucket,
+        'name': path,
     }
 
+    if not os.path.isdir(fs_path):
```

Comment:
Can you add a comment on why we don't send these for directories. Is it ok to generalize ?

Reply:
added a comment.